doc/application/index.rst

```diff
@@ -982,7 +977,7 @@ Create a Debugger Configuration
    - In the SVD Path tab:
 
      - File path: :file:`<zephyr
-       base>\\ext\\hal\\nxp\\mcux\\devices\\MK64F12\\MK64F12.xml`
+       base>\\modules\\hal\\nxp\\mcux\\devices\\MK64F12\\MK64F12.xml`
```

Comment:
I think this should be `workspace root` or `west topdir` instead of `base`, but no big deal